Cargo.toml

```diff
@@ -272,6 +272,7 @@ lto = 'fat'
 debug = false
 incremental = false
 overflow-checks = false
+rustflags = ["-C", "target-cpu=native"]
```

Comment:
this flag is not intended to use on shareable releases since it binds the build to the cpu it was built on. It makes sense only to compile something on your own machine and get the performance improve (although I saw actually a degradation in benchmarks when I tested other project)

Reply:
Thanks. I was trying something out.